bottery/platform/telegram.py

```diff
@@ -51,8 +51,8 @@ class TelegramUser(User):
     https://core.telegram.org/bots/api#user
     '''
     def __init__(self, sender):
-        self.id = sender['id']
-        self.first_name = sender['first_name']
+        self.id = sender.get('id')
+        self.first_name = sender.get('first_name')
```

Comment:
the `id` and `first_name` are both not optional, so, it'll always come from the Telegram API.